packages/core/test/lib/commands/obtain.js

```diff
@@ -2,14 +2,13 @@ const assert = require("chai").assert;
 const command = require("../../../lib/commands/obtain");
 const sinon = require("sinon");
 const CompilerSupplier = require("@truffle/compile-solidity").CompilerSupplier;
-let options, done, solc;
+let options, solc;
 
 describe("obtain", () => {
-  describe(".run(options, done)", () => {
+  describe(".run(options)", () => {
```

Comment:
[Mocha says](https://mochajs.org/#arrow-functions) to use `function () {}` not `() => {}`:

```suggestion
  describe(".run(options)", function () {
```